### ISSUE TITLE
MSSQL Database with hyphen in the name

### ISSUE THREAD

**Issue by tyler-brandt** [Kononnable/typeorm-model-generator#273]
I am attempting to run typeorm-model-generator on an mssql database that contains a hyphen in the name and I am receiving the following error:

```
typeorm-model-generator@0.4.2
[9:58:09 AM] Starting creation of model classes.
tedious deprecated The default value for `config.options.enableArithAbort` will change from `false` to `true` in the next major version of `tedious`. Set the value to `true` or `false` explicitly to silence this message. ../../../../../../../../usr/local/lib/node_modules/typeorm-model-generator/node_modules/mssql/lib/tedious/connection-pool.js:61:23
(node:18234) UnhandledPromiseRejectionWarning: RequestError: Incorrect syntax near '-'.
    at handleError (/usr/local/lib/node_modules/typeorm-model-generator/node_modules/mssql/lib/tedious/request.js:366:15)
    at Connection.emit (events.js:223:5)
    at Parser.<anonymous> (/usr/local/lib/node_modules/typeorm-model-generator/node_modules/tedious/lib/connection.js:832:12)
    at Parser.emit (events.js:223:5)
    at Parser.<anonymous> (/usr/local/lib/node_modules/typeorm-model-generator/node_modules/tedious/lib/token/token-stream-parser.js:37:14)
    at Parser.emit (events.js:223:5)
    at addChunk (/usr/local/lib/node_modules/typeorm-model-generator/node_modules/readable-stream/lib/_stream_readable.js:298:12)
    at readableAddChunk (/usr/local/lib/node_modules/typeorm-model-generator/node_modules/readable-stream/lib/_stream_readable.js:280:11)
    at Parser.Readable.push (/usr/local/lib/node_modules/typeorm-model-generator/node_modules/readable-stream/lib/_stream_readable.js:241:10)
    at Parser.Transform.push (/usr/local/lib/node_modules/typeorm-model-generator/node_modules/readable-stream/lib/_stream_transform.js:139:32)
(node:18234) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:18234) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

The database in question has a name like `ABC-DEF` and I am running typeorm-model-generator with the following command:

```
typeorm-model-generator \
        -e 'mssql' \
        -h 'my_db_ip_address' \
        -p '1433' \
        -o 'src/entities' \
        -d 'ABC-DEF' \
        -u 'my_username' \
        -x 'my_password'
```

I am currently attempting to change the name of my database to something like: `ABCDEF` and I will update this issue if that fixes the problem.

EDIT: Renaming the database fixed the issue

**Comment by BorisKotlyarov** [comment]
I have a similar problem

```bash
typeorm-model-generator@0.4.2
[12:20:32 PM] Starting creation of model classes.
tedious deprecated The default value for `config.options.enableArithAbort` will change from `false` to `true` in the next major version of `tedious`. Set the value to `true` or `false` explicitly to silence this message. ../../.npm-global/lib/node_modules/typeorm-model-generator/node_modules/mssql/lib/tedious/connection-pool.js:61:23
(node:22960) UnhandledPromiseRejectionWarning: RequestError: Incorrect syntax near '1'.
    at handleError (/home/jsuser/.npm-global/lib/node_modules/typeorm-model-generator/node_modules/mssql/lib/tedious/request.js:366:15)
    at Connection.emit (events.js:315:20)
    at Parser.<anonymous> (/home/jsuser/.npm-global/lib/node_modules/typeorm-model-generator/node_modules/tedious/lib/connection.js:832:12)
    at Parser.emit (events.js:315:20)
    at Parser.<anonymous> (/home/jsuser/.npm-global/lib/node_modules/typeorm-model-generator/node_modules/tedious/lib/token/token-stream-parser.js:37:14)
    at Parser.emit (events.js:315:20)
    at addChunk (/home/jsuser/.npm-global/lib/node_modules/typeorm-model-generator/node_modules/readable-stream/lib/_stream_readable.js:298:12)
    at readableAddChunk (/home/jsuser/.npm-global/lib/node_modules/typeorm-model-generator/node_modules/readable-stream/lib/_stream_readable.js:280:11)
    at Parser.Readable.push (/home/jsuser/.npm-global/lib/node_modules/typeorm-model-generator/node_modules/readable-stream/lib/_stream_readable.js:241:10)
    at Parser.Transform.push (/home/jsuser/.npm-global/lib/node_modules/typeorm-model-generator/node_modules/readable-stream/lib/_stream_transform.js:139:32)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:22960) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:22960) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```
```bash

typeorm-model-generator \
        -e 'mssql' \
        -h 'my_host' \
        -p '1433' \
        -d 'C1 \
        -u 'my_username' \
        -x 'my_password'
```

**Comment by j0ruge** [comment]
Same problem here!! ✋🏻

**Comment by Kononnable** [comment]
Implemented in `typeorm-model-generator@0.4.4`
